### ISSUE TITLE
KDE, pandas, and hue

### ISSUE THREAD

**Issue by TDaltonC** [mwaskom/seaborn#595]
Coming from R and ggplot(), I have repeatedly found myself wanting to write this:

``` python
sns.kdeplot(data = DateFrame, x = 'column1', hue = 'column2')
```

expecting to get something like [one of these](https://www.stanford.edu/~mwaskom/software/seaborn/_images/plotting_distributions_34_0.png).

And then I look through the docs to see what I actually have to do to get a plot that looks like that and it seems very cumbersome. Not at all what I would want from a quick-plot/data-exploration tool. 

Is this a syntax that you would be interested in supporting or is there something deeply un-seaborn about it?

**Comment by TomAugspurger** [comment]
@TDaltonC FYI I'll typically do something like

```python
gr = df.groupby('column2').column1
for label, arr in gr:
    sns.kdeplot(arr, label=label, shade=True
```

which isn't too bad.

**Comment by mwaskom** [comment]
Yes something like this will probably happen, depending on how upset people get about the changes to the categorical plots in 0.6.

In addition to Tom's code, you could also do

```python
sns.FacetGrid(data, hue="column2").map(sns.kdeplot, "column1")
```

Which is kind of even more ggplotty.

**Comment by mwaskom** [comment]
Closed in #2104 with the addition of `hue` to `kdeplot`
